language/move-vm/integration-tests/src/tests/bad_storage_tests.rs

```diff
@@ -82,7 +82,10 @@ fn test_malformed_resource() {
     m.serialize(&mut blob).unwrap();
     storage.publish_or_overwrite_module(m.self_id(), blob);
 
-    let vm = MoveVM::new();
+    let vm = MoveVM::new(move_stdlib::natives::all_natives(
+        AccountAddress::from_hex_literal("0x1").unwrap(),
```

Comment:
feels like we should have a constant for this somewhere

Reply:
Strictly speaking this test doesn't care about the exact address used. It just needs to access the signer module. The greater problem here however, is that I'm rolling my own signer module instead of using the one from std. Let me try to get this fixed in a separate PR.